packages/pyright-internal/src/analyzer/pyTypedUtils.ts

```diff
@@ -16,18 +16,32 @@ export interface PyTypedInfo {
     isPartiallyTyped: boolean;
 }
 
+/**
```

Comment:
Nit: Please be consistent with other function comments used throughout the pyright code base and use `//` comments rather than block comments. Same below.